### DISCUSSION TITLE
Walmart.com Support

### DISCUSSION THREAD

**Discussion by moblsu** [moralmunky/Home-Assistant-Mail-And-Packages#806] [Ideas]
Is there a method by which Walmart.com shipments would be trackable? Though they are not actually shipping most items (those coming through Fedex or other carriers) I do get their emails with estimated arrival date, tracking number, etc. If it would be helpful, I can forward examples of those emails.

**Comment by firstof9** [comment]
Do they have a general "out for delivery" and "delivered" email?

**Reply by moblsu** [reply]
Sorry, I replied to an email and not coming in here. So, you might not have seen it:

Standard emails all from "help at walmart.com”. Other bullet points are standard information contained within the body of each email:

```
*	Subject: “👍 Your order was delivered”
   *	“Order date: Wed, Feb 15, 2023”
   *	“Order 2#####7-3######4 was delivered”
   *	Qty: “1 item”
   *	Images of items
*	Subject: “Shipped: items from order #2#####7-3######4” (“#” & 7 digits & “-“ & 7 digits)
   *	“Order date: Wed, Jan 18, 2023”
   *	“Order number: 2#####7-3######4”
   *	Address
   *	Qty and arrival: “1 item arrives by end of day Fri, Jan 20, 2023”
   *	Tracking Information: “Fedex tracking number 62338#######” (hyperlink to tracking information)
   *	Images of items within shipment
*	Subject: “👍 Some of your items were delivered” (thumbs up gif is before the text, if that does not come through here)
   *	Order Date: “Order date: Wed, Jan 18, 2023”
   *	Order Number: “Order 2#####7-3######4 was delivered” where the number is a hyperlink to the order
   *	Number of Items: “1 item”
*	Subject: “Order update: your delivery is delayed”
   *	“Order date: Sun, Feb 5, 2023”
   *	“Order number: 2#####7-3######4”
   *	Qty of items delayed: “18 items delayed”
   *	Images, descriptions, and price of 4 of items delayed
*	Subject: “All set: we canceled your order for you”
   *	“Order date: Sun, Feb 5, 2023”
   *	Order number: 2#####7-3######4”
   *	Qty of items canceled: “18 items canceled”
   *	Images, descriptions, and price of 4 of the items canceled
 ```
I can send you examples of the emails if you need exact formatting.

**Reply by firstof9** [reply]
I'll see what I can do.

**Comment by moblsu** [comment]
Standard emails all from ***@***.***”. Other bullet points are standard information contained within the body of each email:

*	Subject: “👍 Your order was delivered” (thumbs up gif is before the text, if that does not come through here)

*	“Order date: Wed, Feb 15, 2023”
*	“Order 2#####7-3######4 was delivered”
*	Qty: “1 item”
*	Images of items

*	Subject: “Shipped: items from order #2#####7-3######4” (“#” & 7 digits & “-“ & 7 digits)

*	“Order date: Wed, Jan 18, 2023”
*	“Order number: 2#####7-3######4”
*	Address
*	Qty and arrival: “1 item arrives by end of day Fri, Jan 20, 2023”
*	Tracking Information: “Fedex tracking number 62338#######” (hyperlink to tracking information)
*	Images of items within shipment

*	Subject: “👍 Some of your items were delivered” (thumbs up gif is before the text, if that does not come through here)

*	Body includes:

*	Order Date: “Order date: Wed, Jan 18, 2023”
*	Order Number: “Order 2#####7-3######4 was delivered” where the number is a hyperlink to the order
*	Number of Items: “1 item”

*	Subject: “Order update: your delivery is delayed”

*	“Order date: Sun, Feb 5, 2023”
*	“Order number: 2#####7-3######4”
*	Qty of items delayed: “18 items delayed”
*	Images, descriptions, and price of 4 of items delayed

*	Subject: “All set: we canceled your order for you”

*	“Order date: Sun, Feb 5, 2023”
*	Order number: 2#####7-3######4”
*	Qty of items canceled: “18 items canceled”
*	Images, descriptions, and price of 4 of the items canceled

**Comment by firstof9** [comment]
This was implemented in the latest releases.

**Comment by moblsu** [comment]
Thanks, I'll check them out.
